frontend/components/Accounts/Forms/SelectInput.tsx

```diff
@@ -9,10 +9,21 @@ interface SelectOption {
     name: string;
 }
 
-const toSelectOptions = (options) =>
-    options.map((obj) => ({ value: obj.name, label: obj.name }));
+interface FormOption {
```

Comment:
On second thought, I think we should name this OptionValue instead, to disambiguate from the above. For context, SelectOption is the info we get from the backend and what you call FormOption is how we are displaying a particular option.